src/chrome/content/rules/Grsm.io.xml

```diff
@@ -0,0 +1,15 @@
+<!--
+	Wildcard certificate and DNS records:
+		- grsm.io
+-->
+
+<ruleset name="Grsm.io">
+	<target host="grsm.io" />
+	<target host="*.grsm.io" />
+		<test url="http://www.grsm.io/" />
+		<test url="http://a.grsm.io/" />
+		<test url="http://b.grsm.io/" />
+
+	<rule from="^http://([\w-]+\.)?grsm\.io/"
+		to="https://$1grsm.io/" />
```

Comment:
Keep it on one line.

Reply:
I prefer to only one-line `<rule>`s when it is a global redirect for all targets.

Reply:
@injust Got it.